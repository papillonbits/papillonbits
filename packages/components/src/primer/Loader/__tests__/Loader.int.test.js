import { heading, text, label, button } from './Loader.int.story'

describe('<Loader />', () => {
  describe('Render', () => {
    test('must match heading()', () => {
      expect(global.renderToJSON(heading())).toMatchSnapshot()
    })

    test('must match text()', () => {
      expect(global.renderToJSON(text())).toMatchSnapshot()
    })

    test('must match label()', () => {
      expect(global.renderToJSON(label())).toMatchSnapshot()
    })

    test('must match button()', () => {
      expect(global.renderToJSON(button())).toMatchSnapshot()
    })
  })
})
