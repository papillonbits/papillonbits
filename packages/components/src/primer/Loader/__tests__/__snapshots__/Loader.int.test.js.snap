// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Render must match button() 1`] = `
<button
  aria-disabled="true"
  className="btn mt-3"
  type="button"
>
  <span>
    Loading
  </span>
  <span
    className="AnimatedEllipsis"
  />
</button>
`;

exports[`<Loader /> Render must match heading() 1`] = `
<h2
  className={null}
>
  <span>
    Loading
  </span>
  <span
    className="AnimatedEllipsis"
  />
</h2>
`;

exports[`<Loader /> Render must match label() 1`] = `
<span
  className="Label bg-blue mt-3"
>
  <span>
    Loading
  </span>
  <span
    className="AnimatedEllipsis"
  />
</span>
`;

exports[`<Loader /> Render must match text() 1`] = `
<span
  className="branch-name mt-2"
>
  <span>
    Loading
  </span>
  <span
    className="AnimatedEllipsis"
  />
</span>
`;
